### ISSUE TITLE
Column alias produces models that do not compile

### ISSUE THREAD

**Issue by jalan** [volatiletech/sqlboiler#883]
### What version of SQLBoiler are you using (`sqlboiler --version`)?

`v4.4.0`

### What is your database and version (eg. Postgresql 10)

`postgres 13.1`

### If this happened at generation time what was the full SQLBoiler command you used to generate your models? (if not applicable leave blank)

`sqlboiler psql`

### Please provide a relevant database schema so we can replicate your issue (Provided you are comfortable sharing this)

```
CREATE TABLE gpu (
    gpu_id serial PRIMARY KEY
);
CREATE TABLE gpu_thing (
    gpu_thing_id serial PRIMARY KEY,
    gpu_id integer REFERENCES gpu (gpu_id)
);
```

### Further information. What did you do, what did you expect?

I added this alias for one of my columns in `sqlboiler.toml`:

```
[aliases.tables.gpu.columns]
gpu_id = "GPUID"
```

Model generation succeeds, but the result does not compile, because some of the generated code uses the alias and some doesn't:

```
models/gpu.go:409:29: object.GpuID undefined (type *Gpu has no field or method GpuID, but does have GPUID)
models/gpu.go:423:27: obj.GpuID undefined (type *Gpu has no field or method GpuID, but does have GPUID)
```

I believe I located the issue in [templates/09_relationship_to_many_eager.go.tpl](https://github.com/volatiletech/sqlboiler/blob/master/templates/09_relationship_to_many_eager.go.tpl). I can prepare a PR, but I'm not sure which version of `go-bindata` I need to match yours. I tried both the latest release and the one listed in your `go.mod` and they both create unrelated changes.

I have a minimal working example of the issue [here](https://github.com/jalan/sqlboiler_bug/), including the broken models.

**Comment by aarondl** [comment]
I have updated go-bindata to the latest published version (v3.22.0) and pushed the changes it made to the dev branch. Hopefully it won't cause you anymore headaches.

**Comment by jalan** [comment]
Fixed in #885
